mempool/client.go

```diff
@@ -0,0 +1,49 @@
+package mempool
+
+import (
+	"strconv"
+	"unsafe"
+
+	mpproto "github.com/tendermint/tendermint/proto/tendermint/mempool"
+)
+
+type Client struct {
+	mp Mempool
+}
+
+func NewMempoolClient(memAddress string) *Client {
+	ptrInt, _ := strconv.ParseUint(memAddress, 10, 64)
+	ptrVal := uintptr(ptrInt)
+	ptr := unsafe.Pointer(ptrVal)
+	memPtr := (*Mempool)(ptr)
```

Comment:
we probably want to make sure type cast succeeded or not, consider using `ptr.(*Mempool)`?

Reply:
Seems that cannot use type assertion on non-interface type?

Reply:
i see